FashionEditor/FashionEditor.csproj

```diff
@@ -1,5 +1,14 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
+  <ItemGroup>
```

Comment:
Just remove the ItemGroup with None since these files are included as embedded resources.

Reply:
I have remove the ItemGroup with None